descriptors/mysql/debezium-mysql-1.9.0.Alpha1.json

```diff
@@ -259,336 +131,13 @@
-          "default" : 10000,
-          "type" : "integer",
-          "x-name" : "snapshot.lock.timeout.ms",
-          "x-category" : "CONNECTOR_SNAPSHOT"
-        },
-        "snapshot.max.threads" : {
-          "format" : "int32",
-          "title" : "Snapshot maximum threads",
-          "description" : "The maximum number of threads used to perform the snapshot.  Defaults to 1.",
-          "default" : 1,
-          "type" : "integer",
-          "x-name" : "snapshot.max.threads",
-          "x-category" : "CONNECTOR_SNAPSHOT"
-        },
-        "snapshot.select.statement.overrides" : {
-          "title" : "List of tables where the default select statement used during snapshotting should be overridden.",
-          "description" : " This property contains a comma-separated list of fully-qualified tables (DB_NAME.TABLE_NAME) or (SCHEMA_NAME.TABLE_NAME), depending on thespecific connectors. Select statements for the individual tables are specified in further configuration properties, one for each table, identified by the id 'snapshot.select.statement.overrides.[DB_NAME].[TABLE_NAME]' or 'snapshot.select.statement.overrides.[SCHEMA_NAME].[TABLE_NAME]', respectively. The value of those properties is the select statement to use when retrieving data from the specific table during snapshotting. A possible use case for large append-only tables is setting a specific point where to start (resume) snapshotting, in case a previous snapshotting was interrupted.",
-          "type" : "string",
-          "x-name" : "snapshot.select.statement.overrides",
-          "x-category" : "CONNECTOR_SNAPSHOT"
-        },
-        "include.schema.changes" : {
-          "title" : "Include database schema changes",
-          "description" : "Whether the connector should publish changes in the database schema to a Kafka topic with the same name as the database server ID. Each schema change will be recorded using a key that contains the database name and whose value include logical description of the new schema and optionally the DDL statement(s).The default is 'true'. This is independent of how the connector internally records database history.",
-          "default" : true,
-          "type" : "boolean",
-          "x-name" : "include.schema.changes",
-          "x-category" : "CONNECTOR"
-        },
-        "tombstones.on.delete" : {
-          "title" : "Change the behaviour of Debezium with regards to delete operations",
-          "description" : "Whether delete operations should be represented by a delete event and a subsquenttombstone event (true) or only by a delete event (false). Emitting the tombstone event (the default behavior) allows Kafka to completely delete all events pertaining to the given key once the source record got deleted.",
-          "default" : true,
-          "type" : "boolean",
-          "x-name" : "tombstones.on.delete",
-          "x-category" : "CONNECTOR"
-        },
-        "decimal.handling.mode" : {
```

Comment:
@gunnarmorling as this is not in the meeting notes, I think it should be included?